x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -212,7 +279,12 @@ export const FindingsRuleFlyout = ({
         </EuiTabs>
       </EuiFlyoutHeader>
       <EuiFlyoutBody key={tab.id}>
-        <FindingsTab tab={tab} findings={findings} />
+        {!isNativeCspFinding(finding) && ['overview', 'rule'].includes(tab.id) && (
```

Comment:
btw we don't check if some fields are missing, we kind of assume that in 3rd party integrations, some will be missing. Later we might end up in a situation when 3rd party support is so good that it provides all of the fields we rely on here. Do we have a plan to make the logic more granular or remove this at some point?

Reply:
you are right, but i did not think we should actually check every field now. we can definitely create a hook that is solely responsible for checking the existence of all used fields, but it will require some maintenance each time we add/remove fields from the flyout it self. i was not sure if it was the time to do such thing so for now i did not implement that.

Do you think its worth creating a ticket for that at this stage?

Reply:
I think we can create a GitHub issue to track that, it's definitely not a recent future